includes/Abstracts/DokanRESTController.php

```diff
@@ -185,7 +185,9 @@ public function delete_item( $request ) {
         $response = rest_ensure_response( $data );
 
         // If we're forcing, then delete permanently.
-        $object->delete( true );
```

Comment:
instead of checking for true value, send the argument of this method as $request['force']

Reply:
Done!